src/main/java/hwr/oop/List.java

```diff
@@ -64,6 +83,30 @@ public void writeToJSON(String fileName) {
         }
     }
 
+    public void updateBuckets(){
```

Comment:
gibt es dafür vielleicht eine coole java Funktion mit der man schneller suchen kann?

Reply:
da bin ich mir nicht sicher ich schaue mal nach

Reply:
habe das jetzt in addBucket und edit Bucket aufgeteilt